apps/system/style/chrome/chrome.css

```diff
@@ -591,12 +596,12 @@ html[dir="rtl"] .light .controls .forward-button:active {
 
 html[dir="rtl"] .chrome.maximized .title[data-ssl="broken"],
 html[dir="rtl"] .chrome.maximized .title[data-ssl="secure"] {
-  background-position: calc(100% - 0.1rem) 0.1rem;
+  background-position: calc(100% - 0.1rem) calc(50% - 0.1rem);
```

Comment:
you can use the not-that-new 4-values syntax: `background-position: right 0.1rem bottom 0.1rem` to make it more readable. Hopefully one day we'll be able to use start/end in addition to left/right :)